oak_containers_orchestrator_client/README.md

```diff
@@ -0,0 +1,4 @@
+# Orchestrator Client
```

Comment:
Maybe it makes sense to add the README to `Oak Containers Orchestrator` because I kind think of `oak_containers_launcher` as the "interface". Does that make sense?

(Analogous for the Oak Containers Launcher).

Reply:
I'm not sure I understand, could you rephrase?